test/unit/batching/vendor-lines/get-payment-type.test.js

```diff
@@ -0,0 +1,108 @@
+const { BPS, CS, FDMR, LumpSums, SFI, SFIP, VetVisits, IMPS, ES } = require('../../../../app/constants/schemes')
+const { getPaymentType } = require('../../../../app/batching/vendor-lines/get-payment-type')
+
+describe('get payment type tests', () => {
+  let paymentType
+
+  test('return 1 when schemeId is CS and paymentType is 1', () => {
+    paymentType = '1'
+    const result = getPaymentType(CS, paymentType)
+    expect(result).toBe(1)
+  })
+
+  test('return 0 when schemeId is CS and paymentType is 0', () => {
+    paymentType = '0'
+    const result = getPaymentType(CS, paymentType)
+    expect(result).toBe(0)
+  })
+
+  test('return 0 when schemeId is CS and paymentType is 2', () => {
+    paymentType = '0'
```

Comment:
Should this not be defined as 2?

Reply:
This test was already written but agree it's wrong.  Have corrected it.